index.html

```diff
@@ -0,0 +1,699 @@
+            </h2>
+            <p>
+              by Rachel Kaser<i class="fa fa-clock" aria-hidden="true"></i>1
+              hour ago<i class="fa fa-share-alt" aria-hidden="true"></i>8
+            </p>
+          </a>
+        </div>
+      </section>
+
+      <div>
+        <h2 class="section-tittle">
+          Latest news <img src="./images/next-page.png" alt="next-page" />
+        </h2>
+      </div>
+      <section class="grid latest-news">
+        <div>
+          <div class="latest-news-1 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              This app helps you create the resume, portfolio and more to get
+              your next job
+            </h3>
+            <p>TNW Deals • 6 minutes ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-2 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              Here are games you can play with <br />friends online while stuck
+              at home
+            </h3>
+            <p>Rachel Kaser • 14 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-3 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>AI is searching for unexploded Vietnam <br />War bombs</h3>
+            <p>Thomas Macaulay • 14 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-4 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>Check out these lesser-known Python <br />features</h3>
+            <p>James Briggs • 6 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-5 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>How to watch Huawei's P40 launch live <br />on March 26</h3>
+            <p>Napier Lopez • 17 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-6 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              Bezos dumped $3.4B in Amazon shares <br />before coronavirus
+              outbreak
+            </h3>
+            <p>David Canellis • 17 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-7 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              How to protect AI systems against <br />image-scaling attacks
+            </h3>
+            <p>Ben Dickson • 3 hours ago</p>
+          </div>
+        </div>
+        <div>
+          <div class="latest-news-8 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              5 tips on how web devs can survive <br />(and thrive) during a
+              recession
+            </h3>
+            <p>Andrea Hak • 2 hour Ago</p>
+          </div>
+        </div>
+      </section>
```

Comment:
**Latest news section**

- [ ]  It aligns 3 articles per row ([link to page](https://thenextweb.com/)) in medium screens in the original page but on your page, it contains 4 articles per row. Also, a horizontal scroll appears. Please kindly fix it.

![image](https://user-images.githubusercontent.com/60184961/91213074-bf6cd200-e719-11ea-8258-5863f1b77a23.png)